packages/api/src/hooks/useCryptoTransactions.ts

```diff
@@ -58,6 +58,45 @@ const getStatusName = (status_code: TModifiedTransaction['status_code']) => {
     }
 };
 
+const getStatusDescription = (
+    transaction_type: TModifiedTransaction['transaction_type'],
+    status_code: TModifiedTransaction['status_code']
+) => {
+    switch (status_code) {
+        // deposit-specific:
+        case 'CONFIRMED':
+            return 'Your deposit is successful.';
+        case 'PENDING':
+            return "We've received your request and are waiting for more blockchain confirmations.";
+        // withdrawal-specific:
+        case 'CANCELLED':
+            return "You've cancelled your withdrawal request.";
+        case 'LOCKED':
+            return "We're reviewing your withdrawal request. You may still cancel this transaction if you wish.\nOnce we start processing, you won't be able to cancel.";
+        case 'PERFORMING_BLOCKCHAIN_TXN':
+            return "We're sending your request to the blockchain.";
+        case 'PROCESSING':
+            return "We're awaiting confirmation from the blockchain.";
+        case 'REJECTED':
+        case 'REVERTED':
+            return "Your withdrawal is unsuccessful. We've sent you an email with more information.";
+        case 'REVERTING':
+        case 'VERIFIED':
+            return "We're processing your withdrawal.";
+        case 'SENT':
+            return 'Your withdrawal is successful.';
+        // both:
+        case 'ERROR':
+            return `Your ${
+                transaction_type === 'deposit' ? 'deposit' : 'withdrawal'
```

Comment:
```suggestion
                transaction_type
```

Reply:
Yea I know
But is it ok to just put the API value to user message directly like this?

Reply:
We know the types though so this should be ok